utils/kamctl/mysql/ims_usrloc_pcscf-create.sql

```diff
@@ -1,51 +1,52 @@
-  `rx_session_id` varchar(256) DEFAULT NULL,
-  `reg_state` tinyint(4) DEFAULT NULL,
-  `expires` datetime DEFAULT '2030-05-28 21:32:15',
-  `service_routes` varchar(2048) DEFAULT NULL,
-  `socket` varchar(64) DEFAULT NULL,
-  `public_ids` varchar(2048) DEFAULT NULL,
-  `security_type` int(11) DEFAULT NULL,
-  `protocol` int(10) DEFAULT NULL,
-  `mode` char(10) DEFAULT NULL,
-  `ck` varchar(100) DEFAULT NULL,
-  `ik` varchar(100) DEFAULT NULL,
-  `ealg` char(20) DEFAULT NULL,
-  `ialg` char(20) DEFAULT NULL,
-  `port_pc` int(11) unsigned DEFAULT NULL,
-  `port_ps` int(11) unsigned DEFAULT NULL,
-  `port_uc` int(11) unsigned DEFAULT NULL,
-  `port_us` int(11) unsigned DEFAULT NULL,
-  `spi_pc` int(11) unsigned DEFAULT NULL,
-  `spi_ps` int(11) unsigned DEFAULT NULL,
-  `spi_uc` int(11) unsigned DEFAULT NULL,
-  `spi_us` int(11) unsigned DEFAULT NULL,
-  `t_security_type` int(11) DEFAULT NULL,
-  `t_port_pc` int(11) unsigned DEFAULT NULL,
-  `t_port_ps` int(11) unsigned DEFAULT NULL,
-  `t_port_uc` int(11) unsigned DEFAULT NULL,
-  `t_port_us` int(11) unsigned DEFAULT NULL,
-  `t_spi_pc` int(11) unsigned DEFAULT NULL,
-  `t_spi_ps` int(11) unsigned DEFAULT NULL,
-  `t_spi_uc` int(11) unsigned DEFAULT NULL,
-  `t_spi_us` int(11) unsigned DEFAULT NULL,
-  `t_protocol` char(5) DEFAULT NULL,
-  `t_mode` char(10) DEFAULT NULL,
-  `t_ck` varchar(100) DEFAULT NULL,
-  `t_ik` varchar(100) DEFAULT NULL,
-  `t_ealg` char(20) DEFAULT NULL,
-  `t_ialg` char(20) DEFAULT NULL,
-  PRIMARY KEY (`id`),
-  KEY `aor` (`aor`)
+    `id` INT(10) UNSIGNED AUTO_INCREMENT PRIMARY KEY NOT NULL,
+    `domain` VARCHAR(64) NOT NULL,
+    `aor` VARCHAR(255) NOT NULL,
+    `host` VARCHAR(100) NOT NULL,
+    `port` INT(10) NOT NULL,
+    `received` VARCHAR(128) DEFAULT NULL,
+    `received_port` INT(10) UNSIGNED DEFAULT NULL,
+    `received_proto` INT(10) UNSIGNED DEFAULT NULL,
+    `path` VARCHAR(512) DEFAULT NULL,
+    `rinstance` VARCHAR(255) DEFAULT NULL,
+    `rx_session_id` VARCHAR(256) DEFAULT NULL,
+    `reg_state` TINYINT(4) DEFAULT NULL,
+    `expires` DATETIME DEFAULT '2030-05-28 21:32:15',
+    `service_routes` VARCHAR(2048) DEFAULT NULL,
+    `socket` VARCHAR(64) DEFAULT NULL,
+    `public_ids` VARCHAR(2048) DEFAULT NULL,
+    `security_type` INT(11) DEFAULT NULL,
+    `protocol` INT(10) DEFAULT NULL,
+    `mode` VARCHAR(10) DEFAULT NULL,
+    `ck` VARCHAR(100) DEFAULT NULL,
+    `ik` VARCHAR(100) DEFAULT NULL,
+    `ealg` VARCHAR(20) DEFAULT NULL,
+    `ialg` VARCHAR(20) DEFAULT NULL,
+    `port_pc` INT(11) UNSIGNED DEFAULT NULL,
+    `port_ps` INT(11) UNSIGNED DEFAULT NULL,
+    `port_uc` INT(11) UNSIGNED DEFAULT NULL,
+    `port_us` INT(11) UNSIGNED DEFAULT NULL,
+    `spi_pc` INT(11) UNSIGNED DEFAULT NULL,
+    `spi_ps` INT(11) UNSIGNED DEFAULT NULL,
+    `spi_uc` INT(11) UNSIGNED DEFAULT NULL,
+    `spi_us` INT(11) UNSIGNED DEFAULT NULL,
+    `t_security_type` INT(11) DEFAULT NULL,
+    `t_protocol` INT(10) DEFAULT NULL,
```

Comment:
This was char(5) before, maybe we want to convert to VARCHAR instead of INT?

Reply:
yes, copy paste error from my side. Thanks!